pkg/convert/convert.go

```diff
@@ -8,3 +10,15 @@ func convertToMessageFormatSingular(message string) string {
 
 	return "{" + message + "}"
 }
+
+// convertFromMessageFormatStrToStr function replaces '{' and '}' only if message starts with '{' and ends with '}'.
```

Comment:
This seems like it could be a generic function e.g. -
removeBracketsFromString()